app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -149,32 +163,34 @@ public void onClick(View view) {
         });
 
         // Click listeners for changing activity content
-        joystickContentBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (headsetIsConnected && carIsConnected) {
-                    joystickContent.setVisibility(View.VISIBLE);
-                    eegContent.setVisibility(View.GONE);
-
-                    joystickContentBtn.setVisibility(View.GONE);
-                    eegContentBtn.setVisibility(View.VISIBLE);
-
-                    if (eegActive) {
-                        eegActive = false;
-                        stopEeg();
-                        stopGyro();
-                    }
-                }
-                else {
-                    // do nothing
-                }
-            }
-        });
 
-        eegContentBtn.setOnClickListener(new View.OnClickListener() {
+      joystickContentBtn.setOnClickListener(new View.OnClickListener() {
+          @Override
+          public void onClick(View v) {
+           //   if (headsetIsConnected && carIsConnected) {
+//                    //joystickContent.setVisibility(View.VISIBLE);
+//                    eegContent.setVisibility(View.GONE);
+//
+//                    joystickContentBtn.setVisibility(View.GONE);
+//                    eegContentBtn.setVisibility(View.VISIBLE);
+//
+//                    if (eegActive) {
+//                        eegActive = false;
+//                        stopEeg();
+//                        stopGyro();
+//                    }
+//                }
+//                else {
+//                    // do nothing
+//                }
+                  SecondActivity.start(MainActivity.this);
```

Comment:
The reason I didn't add a second activity in the original UI was because I couldn't get the header to work when the activities were split up into two. Have you gotten this working when implementing a second activity?

The idea is that the content of the "body" of the UI changes, but the header of the ui is "static".

Reply:
no, have same problem. will try to remove second activity into main activity.

Reply:
You don't need to put the contents of activity_second in activity_main. You can just edit the content_joystick file instead, as that file is already included in the activity_main file.